### PR TITLE
Actually add rules for the-new-hotness.

### PR DESCRIPTION
**Pull request by ralphbean**
These were forgotten in #28 -- whoops!

### CODE REVIEW

File: fmn/rules/hotness.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+from fmn.lib.hinting import hint, prefixed as _
+
+
+@hint(topics=[_('hotness.update.bug.file')])
+def hotness_bug_file(config, message):
+    """ New bugs filed for upstream releases
+
+    There is a constellation of services involvedfoonotifying packagers of
```
Comment (pypingou) [inline fmn/rules/hotness.py]: ``involvedfoonotifying``?

File: fmn/rules/hotness.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    `new upstream releases
+    <https://fedoraproject.org/wiki/Upstream_release_monitoring>`_.
+    Adding this rule will trigger notifications from the component called
+    `the-new-hotness <https://github.com/fedora-infra/the-new-hotness>`_ when
```
Comment (pypingou) [inline fmn/rules/hotness.py]: Maybe to be updated with the wiki page when ready?

**Comment by pypingou** [comment]
2 comments, otherwise :+1:

File: fmn/rules/hotness.py (hunk 1 above)
Comment (ralphbean) [inline fmn/rules/hotness.py]: no idea.  will fix :)

File: fmn/rules/hotness.py (hunk 2 above)
Comment (ralphbean) [inline fmn/rules/hotness.py]: What do you mean by this?

Comment (pypingou) [inline fmn/rules/hotness.py]: When we have the wiki page for upstream_monitoring, I was wondering if that would not be a better place to point to for an explanation of what is the-new-hotness is

Comment (ralphbean) [inline fmn/rules/hotness.py]: Oh!  It's already linked to in the docstring -- see ``new upstream releases`` there.

**Pull request merged**